### PR TITLE
Cross-publish for SBT 1.0

### PR DESCRIPTION
**Pull request by philwills**
At least as far as compilation goes, this seems to work.

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16
```
Comment (adamnfish) [inline project/build.properties:1]: Should this be at v1 now so it gets built with 2.12?

File: build.sbt [hunk 2]
```diff
@@ -3,13 +3,18 @@ organization := "com.gu"
 
 sbtPlugin := true
 
-scalaVersion := "2.10.6"
+scalaVersion := "2.12.2"
+sbtVersion in Global := "1.0.0"
+crossSbtVersions := Seq("1.0.0", "0.13.16")
```
Comment (adamnfish) [inline build.sbt]: Is this required?

File: project/build.properties (hunk 1 above)
Comment (philwills) [reply project/build.properties:1]: Not if you want to cross-build. See http://www.scala-sbt.org/1.0/docs/Cross-Build-Plugins.html

File: build.sbt (hunk 2 above)
Comment (philwills) [reply build.sbt]: Possibly not. I think there's a reasonable argument for just saying you have to jump to 1.0, but given it was so little effort, I'm minded to cross-publish until it becomes a pain.

File: build.sbt [hunk 3]
```diff
@@ -3,13 +3,18 @@ organization := "com.gu"
 
 sbtPlugin := true
 
-scalaVersion := "2.10.6"
+scalaVersion := "2.12.2"
```
Comment (mchv) [inline build.sbt]: `2.12.3`

**Pull request merged**